tests/chainer_tests/test_optimizer.py

```diff
@@ -638,79 +629,63 @@ def setUp(self):
             lr = self.loss_scale
             for i in range(2):
                 self.target[i].param._loss_scale = self.loss_scale
+        # TODO(niboshi): Do not use SGD in GradientMethod test
         self.optimizer = chainer.optimizers.SGD(lr)
 
-    def setup_cpu(self):
-        self.optimizer.setup(self.target)
-
-    def setup_gpu(self, device=None):
-        self.target.to_gpu(device)
-        self.optimizer.setup(self.target)
-
-    def setup_chainerx(self, orig_xp):
-        if orig_xp is cuda.cupy:
-            self.target.to_device('cuda:0')
-        else:
-            assert orig_xp is np
-            self.target.to_device('native:0')
-        self.optimizer.setup(self.target)
-
-    def check_update(self):
-        self.optimizer.update()
-        xp = backend.get_array_module(self.target[0].param)
+    def test_update(self, backend_config):
+        if backend_config.device == chainer.get_device('@cupy:1'):
```

Comment:
Seems like this test is executed on Travis.

Reply:
Thanks. Fixed.